tap_linkedin_ads/sync.py

```diff
@@ -518,15 +518,13 @@ def merge_responses(data):
                 full_records[primary_key] = element
     return full_records
 
-def sync_ad_analytics(client, state, start_date, stream_name, path, endpoint_config, data_key, static_params,
+def sync_ad_analytics(client, state, last_datetime, stream_name, path, endpoint_config, data_key, static_params,
                       bookmark_query_field=None, bookmark_field=None, id_fields=None, parent=None, parent_id=None):
     # pylint: disable=too-many-branches,too-many-statements,unused-argument
 
-
-    last_timestamp = get_bookmark(state, stream_name, start_date)
-    last_datetime_dt = strptime_to_utc(last_timestamp)
-
-    window_start_date = last_datetime_dt.date()
+    last_datetime = get_bookmark(state, stream_name, last_datetime)
```

Comment:
Looks good, but what if `last_datetime` is None or it does not exist in `state`. I think we should have some default start_date here in that case